typescript/widgets/package.json

```
@@ -1,16 +1,16 @@
 {
   "name": "@hyperlane-xyz/widgets",
   "description": "Common react components for Hyperlane projects",
-  "version": "8.6.1",
+  "version": "8.6.1-testing",
```

Let's start to put some numbering here, so that we can keep track of it. I suggest to do the same with the other packages :)

```suggestion
  "version": "8.6.1-lukso-dev0",
```